### DISCUSSION TITLE
没人维护了？

### DISCUSSION THREAD

**Discussion by GreatMichaelLee** [kaaass/ZerotierFix#95] [General]
多好的一个项目，可惜了

**Comment by kaaass** [comment]
只有我在维护，而我要忙学业，所以……
目前的情况只能是偶尔修 BUG，新功能是没有时间加了，有空尽量维护了

**Reply by GreatMichaelLee** [reply]
> 只有我在维护，而我要忙学业，所以…… 目前的情况只能是偶尔修 BUG，新功能是没有时间加了，有空尽量维护了

感谢大佬啊，能不能先把核心升到1.14.0，其他功能再说

**Reply by kaaass** [reply]
没问题

**Reply by GreatMichaelLee** [reply]
> 没问题

感谢啊~
